src/items/sxa-content/content/Verticals/Services/Home.yml

```diff
@@ -1,155 +1,190 @@
 ﻿---
-ID: "57f2e85d-d695-4e6c-8c89-947ce5443e2e"
-Parent: "a13c890b-43ea-45e2-8bad-a8838112b83d"
-Template: "5282d418-6c18-46eb-85af-9116cde9e6eb"
+ID: '57f2e85d-d695-4e6c-8c89-947ce5443e2e'
```

Comment:
@chpa7 @sc-nelikostadinova This broke deployment, not sure why it was serialized like this.